package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "webserversnodejs",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/main.js",
+  "scripts": {
+    "preinstall": "cd ./client && npm install",
+    "build:client": "cd ./client && npm run build:prod",
+    "dev": "cross-env NODE_ENV=development APP_PORT=8080 nodemon ./src/main.js",
+    "start": "npm run build:client && cross-env NODE_ENV=production APP_PORT=8080 node ./src/main.js"
```

Comment:
It is better to use .env for storing information like ports and another kind of environments variable. Take a look of how it is done in client app.